indexer.go

```diff
@@ -169,15 +156,57 @@ func (i *Indexer) IndexBlock(blockHeight int) error {
 		return ErrBlockHasNoHash
 	}
 
-	err = i.writer.WriteBlock(convertProviderBlockToBlock(blockOutput))
+	blockTxs, err := getProviderBlockTransactions(blockHeight, i.provider)
+
+	if err != nil {
+		return err
+	}
+
+	relayCount := 0
+
+	for _, tx := range blockTxs {
+		if tx.TxResult.MessageType == "claim" {
+			proofs := tx.StdTx.Msg.Value["total_proofs"].(int)
```

Comment:
You should verify with an ok to see if the value was an int and see how you manage it when it isn't or it doesn't exist. Example:
https://github.com/pokt-foundation/pocket-indexer-lib/blob/master/indexer.go#:~:text=rawToAddress%2C%20ok,%7D

Reply:
Also are we sure this comes as an INT?